README.md

```diff
@@ -6,106 +8,64 @@
 
 1. What is the first step for installing Dart on a Windows machine?
 
-A) Install Homebrew
-B) Download the Dart SDK
-C) Update your PATH
-D) Run Dart Doctor
-
+'C) Update your PATH '
 
 2. Which command verifies the Dart installation on macOS?
 
-A) dart --install
-B) dart --check
-C) dart --verify
 D) dart --version
 
 
 3. What is the next step after downloading and extracting the Flutter SDK on Linux?
 
-A) Install Homebrew
-B) Update your PATH
-C) Run Flutter Doctor
-D) Create a new Flutter project
-
+'C) Update your PATH '
 
 4. What command is used to run a newly created Flutter app?
 
-A) flutter start
-B) flutter build
-C) flutter run
-D) flutter init
+'C) flutter run '
 
 
 #Python Installation
 
```

Comment:
This is the next step after installing flutter sdk

Reply:
B) Update your PATH

Reply:
B) Update your path

Reply:
B. Update your path

Reply:
Update your PATH

Reply:
Update your PATH